phases/16-multi-agent-and-swarms/03-communication-protocols/docs/en.md

```
@@ -1,489 +1,1402 @@
+            },
+            "mediaType": "application/json"
+          }]
+        }
+      ]
+    }
+  }
+}
+```
+
+**Streaming via SSE:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)